samples/Exporters/TestPrometheus.cs

```diff
@@ -31,22 +32,41 @@ internal class TestPrometheus
         internal static object Run()
         {
             var promOptions = new PrometheusExporterOptions() { Url = "http://localhost:9184/metrics/" };
-            Metric<long> metric = new Metric<long>("sample");
-            var promExporter = new PrometheusExporter<long>(promOptions, metric);
+            var promExporter = new PrometheusExporter(promOptions);
+            var simpleProcessor = new UngroupedBatcher(promExporter, TimeSpan.FromSeconds(5));
+            var meter = MeterFactory.Create(simpleProcessor).GetMeter("library1") as MeterSDK;
```

Comment:
here I see processor is configured directly on factory and than meter is created out of it. I would expect more separation of factory construction and meter creation. It will not be how users will use API, correct?

Reply:
Correct. Working on getting it right.